deep_quoridor/src/agents/alphazero/nn_evaluator.py

```diff
@@ -65,9 +65,9 @@ def evaluate(self, game: Quoridor, extra_games_to_evaluate: list[Quoridor] = [])
             # If the game was originally rotated, rotate the resulting back to player 2's perspective
             if g.get_current_player() == Player.TWO:
                 policy_masked[i] = policy_masked[i][self.action_mapping_rotated_to_original]
-            self.cache[all_hashes[i]] = (values[i], policy_masked[i])
+            self.cache[all_hashes[i]] = (values[i][0], policy_masked[i])
```

Comment:
Was this a bug before? Why wasn't it breaking things?

Reply:
I was debugging for this PR and noticed that instead of getting a float for the value I was getting a ndarray with 1 float.  Nothing changed when I did this change, so I don't think it was breaking anything, probably somewhere else it was casted as a float or something.  In any case, this seems the right way to do it.